src/config/urls.py

```diff
@@ -29,6 +29,7 @@
     path("inventory/", include("src.apps.inventory.urls")),
     path("checkout/", include("src.apps.orders.urls")),
     path("thank_you/<int:order_id>/", ThankYouView.as_view(), name="thank_you"),
+    path("__debug__/", include("debug_toolbar.urls")),
```

Comment:
Shoudn`t we add this path if we are in Debug (add it to the block below)?

Reply:
It won't make any difference but for the consistency let's do it )